openmp/runtime/src/kmp_alloc.cpp

```diff
@@ -1563,6 +1563,7 @@ static bool __kmp_is_hwloc_membind_supported(hwloc_membind_policy_t policy) {
   return false;
 #endif
 }
+#endif
```

Comment:
Looks wrong. The `#if` above was already there, so adding an extra `#endif` will definitely break something.